pulp_rpm/tests/functional/constants.py

```diff
@@ -145,17 +145,23 @@
 RPM_MODULES_DEFAULTS_COUNT = 3
 """The number of modules-default present on `RPM_MODULAR_FIXTURE_URL`."""
 
+RPM_MODULES_OBSOLETES_COUNT = 1
+"""The number of module-obsoletes present on `RPM_MODULAR_FIXTURE_URL`."""
```

Comment:
This is pending some changes to the test fixtures, correct?

Let's reach out to the modularity folks and see if they have any test fixtures available, since I'm not really all that satisfied with the ones we have, and we know it causes some problems that the RPMs are shared with the nonmodular fixtures.

Reply:
@pavelpicka This is another topic for the modularity meeting